pkg/venafi/tpp/tpp.go

```diff
@@ -624,25 +630,6 @@ func parseRequestData(b []byte) (data certificateRequestResponse, err error) {
 	return
 }
 
-func parseRetrieveResult(httpStatusCode int, httpStatus string, body []byte) (certificateRetrieveResponse, error) {
```

Comment:
Is it possible to keep these functions? that is with the idea to keep the approach to have the parsing of the results in dedicated functions

Reply:
I reverted my changes and made sure `parseRetrieveResult` stays. Please take another look. Thanks!